### PR TITLE
Fix TwoButtonPage issue on shell

### PR DESCRIPTION
**Pull request by shyunMin**
### Description of Change ###
This PR is to fix TwoButtonPage issue on shell

### Bugs Fixed ###
- Before
before

- After
after

### CODE REVIEW

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/TizenCircleSurfaceEffect.cs [hunk 1]
```diff
@@ -43,7 +44,15 @@ protected override void OnAttached()
         {
             var rect = Control.Geometry;
 
-            _surfaceLayout = new ElmSharp.Layout(Container);
+            if (Container != null)
```
Comment (myroot) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/TizenCircleSurfaceEffect.cs]: Do you know which case Container could be null?

Comment (shyunMin) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/TizenCircleSurfaceEffect.cs]:
```
void OnRegisterEffect(PlatformEffect effect)
{
    effect.SetContainer(Element.Parent == null ? null : Platform.GetRenderer(Element.Parent)?.NativeView);
    effect.SetControl(NativeView);
}
```

https://github.com/xamarin/Xamarin.Forms/blob/112a203f77de4807bf40eb7882b01b77af5fe51f/Xamarin.Forms.Platform.Tizen/Renderers/VisualElementRenderer.cs#L804

In this issue,  `Element.Parent` is `ShellSection`, however there is no renderer registered for `ShellSection` in `Tizen.CircularUI`. So `Container` seems to be set was set to null.

Comment (myroot) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/TizenCircleSurfaceEffect.cs]: no need to use Container, Always use XForms.NativeParent

Comment (shyunMin) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/TizenCircleSurfaceEffect.cs]: Fixed.

**Pull request merged**